src/System.Device.Gpio/Interop/Unix/libgpiod/Interop.gpiod_ctxless_event_loop.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information
+
+using System;
+using System.Device.Gpio;
+using System.Runtime.InteropServices;
+
+internal partial class Interop
+{
+    [DllImport(library, SetLastError = true)]
+    internal static extern int gpiod_ctxless_event_loop(string device, uint offset, 
+        bool active_low, string consumer, ref timespec timeout, 
+        [MarshalAs(UnmanagedType.FunctionPtr)]gpiod_ctxless_event_poll_cb poll_cb, 
+        [MarshalAs(UnmanagedType.FunctionPtr)]gpiod_ctxless_event_handle_cb event_cb, IntPtr data);
+}
+
+struct gpiod_ctxless_event_poll_fd
+{
+    int fd;
```

Comment:
One other error you are hitting in the CI build is that these two members don't seem to be used. I assume that is only because you haven't implemented the Eventing?